### ISSUE TITLE
[BUG] Logger.debug() called on tokenActionHudCoreApiReady creates infinite loop.  

### ISSUE THREAD

**Issue by draconas1** [Larkinabout/fvtt-token-action-hud-core#245]
**Describe the bug**

```javascript
Hooks.once('tokenActionHudCoreApiReady', async (coreModule) => {
    coreModule.api.Logger.debug('badger')
}
```

However The debug method first makes a check for the debug variable.  And int his case it is hitting the `else` condition so calls into `Utils.getSetting('debug')`

```javascript
 static debug (message, data) {
        const isDebug = (game.tokenActionHud) ? game.tokenActionHud.debugSetting : Utils.getSetting('debug')
```

getSetting has a catch block that calls `Logger.debug` so if the setting is not found and the catch runs we end up in an infinite loop looking for the debug flag.

This causes an infinite loop, breaking with a stack overflow

```
ils.js:18 Uncaught (in promise) RangeError: undefined. Maximum call stack size exceeded
[Detected 2 packages: token-action-hud-core, foundry-4e-tools]
    at Logger.debug (utils.js:18:90)
    at Utils.getSetting (utils.js:176:20)
    at Logger.debug (utils.js:18:90)
    at Utils.getSetting (utils.js:176:20)
    at Logger.debug (utils.js:18:90)
 ```

**Steps to reproduce**
Creating a sample hook listener that immediately calls debug should do the trick.  

**Versions:**
- Token Action HUD Core: 1.5.4
- Token Action HUD System: dnd4e
- System: dnd4e
- Foundry VTT: 11.315
- Browser: Foundry App

**Additional context**
Any additional context.

**Comment by draconas1** [comment]
Interestingly I have debug enabled on this, so am unsure why it was hitting the catch block and getting the infinite loop.

**Comment by Larkinabout** [comment]
Fixed in Token Action HUD Core 1.5.6.
